etc/links.profile

```diff
@@ -0,0 +1,51 @@
+# Firejail profile for elinks
+# Description: Text WWW browser
+# This file is overwritten after every install/update
+#
+# Persistent local customizations
+include links.local
+# Persistent global definitions
+# include globals.local
+
+blacklist /tmp/.X11-unix
+
+noblacklist ${HOME}/.links
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.links
+whitelist ${HOME}/.links
+whitelist ${DOWNLOADS}
+include whitelist-common.inc
+include whitelist-var-common.inc
+
+caps.drop all
+ipc-namespace
+machine-id
```

Comment:
`machine-id` tends to break audio on some systems. Does links allow playing videos or audio? If so it's probably better to remove this.

Reply:
You're right. I did not think of a situation that `links` downloads an audio file and passes it to external player - will remove that.

Reply:
sandboxed links wouldn't be able to open vlc (or whatever else) anyway
as long as it can't directly play audio itself those options should stay

Reply:
`# uncomment $$$ (or put 'ignore $$$' in your PROFILE.local)`